roles/ansible-test-inventory/templates/inventory-ios.j2

```diff
@@ -4,6 +4,9 @@
 [ios]
 ios-15.6-2T ansible_port=22 ansible_host={{ hostvars['ios-15.6-2T'].ansible_host }} ansible_user=zuul
 
+[resource:children]
+ios
```

Comment:
this should not be needed

Reply:
So before doing this we were getting consistent failures because of missing inventory.

Reply:
this was used by venv testing, no longer with network-ee.  If you depends-on https://github.com/ansible-collections/cisco.ios/pull/397 you'll see we actually delete it now.

Reply:
okay I'll try with it .